src/event-stream/event-server.ts

```diff
@@ -625,13 +625,13 @@ export async function startEventServer(opts: {
                   unresolvedSubdomain.result.name
                 ),
                 owner: parsedTxt.owner,
-                zonefile_hash: unresolvedSubdomain.result.zonefile_hash,
-                zonefile: attachment.content,
+                zonefile_hash: parsedTxt.zoneFileHash,
+                zonefile: parsedTxt.zoneFile,
                 latest: true,
                 tx_id: unresolvedSubdomain.result.tx_id,
                 index_block_hash: unresolvedSubdomain.result.index_block_hash,
                 canonical: unresolvedSubdomain.result.canonical,
-                parent_zonefile_hash: parsedTxt.zoneFile,
+                parent_zonefile_hash: unresolvedSubdomain.result.zonefile_hash,
```

Comment:
@zone117x  I'm not sure about the `parent_zonefile_hash` for subdomains. Is it the hash of the respective name's zone file?
cc: @lgalabru

Reply:
Yes I think so